src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -126,15 +141,13 @@ internal void AddProvider(string workerDir)
 
                     _logger.LogDebug("Found worker config: {workerConfigPath}", workerConfigPath);
 
-                    // Parse worker config file
-                    string json = File.ReadAllText(workerConfigPath);
-                    JObject workerConfig = JObject.Parse(json);
-                    RpcWorkerDescription workerDescription = workerConfig.Property(WorkerConstants.WorkerDescription).Value.ToObject<RpcWorkerDescription>();
+                    JsonElement workerConfig = GetWorkerConfigJsonElement(workerConfigPath);
```

Comment:
nit (consistency): using `var` in the other variable declarations, but the type name here (even though those are all coming as a result of method calls).  You can switch this.

Reply:
ah, yea, fixed to be consistent.